public/js/actions/user.js

```diff
@@ -127,3 +127,25 @@ export function userSignOut(jquery=$) {
     });
   };
 }
+
+
+export function getBraintreeToken() {
+  console.log('Requesting braintree token');
+  return dispatch => {
+    $.ajax({
+      method: 'post',
+      url: '/api/braintree/token/generate/',
+      context: this,
+    }).then(data => {
+      dispatch({
+        type: actionTypes.RX_BRAINTREE_TOKEN,
```

Comment:
Happy to have ideas on a better name for this - but this seemed to be concise and said what it does.